tests/Session/SessionStoreTest.php

```diff
@@ -75,7 +75,7 @@ public function testSessionInvalidate()
         $oldId = $session->getId();
 
         $session->put('foo', 'bar');
-        $this->assertGreaterThan(0, count($session->all()));
+        $this->assertGreaterThan(0, $session->all());
```

Comment:
This looks like it's testing a different concept.

Yes, technically the test still passes etc - but conceptually we were testing how many session data was returned. Now your just saying if the result of `all()` is not 0 - which is not *really* the same thing...?

Reply:
I guess not because `assertGreaterThan` method takes the second argument and apply to `count` method.